### PR TITLE
Update maximumuploadsizesettings

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Small PR to improve readability. 

- [x] Write shorter sentence.

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)
v10, 11 and 12

### CODE REVIEW

File: 10/umbraco-cms/reference/configuration/maximumuploadsizesettings.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ An example of a configuration could look something like this:
 }
 ```
 
-`MaxRequestLength` is specified in kilobytes, so this configuration would limit requests, and therefore uploaded files, to 2 megabytes, and a maximum query string length of 90 characters.
+`MaxRequestLength` is specified in kilobytes. This configuration would limit requests, and therefore uploaded files, to 2 megabytes, and a maximum query string length of 90 characters.
```
Comment (sofietoft) [inline 10/umbraco-cms/reference/configuration/maximumuploadsizesettings.md:53]: Does it make sense to say "This configuration **will** limit requests ... ", instead?
At least that sounds a little better to me 🤔 What do you think?

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @erikjanwestendorp ! 💪 

I've made a comment - let me know what you think!

File: 10/umbraco-cms/reference/configuration/maximumuploadsizesettings.md (hunk 1 above)
Comment (erikjanwestendorp) [reply 10/umbraco-cms/reference/configuration/maximumuploadsizesettings.md]: @sofietoft Definitely agree!

**Comment by erikjanwestendorp** [comment]
@sofietoft Thaks four your review, just updated the PR with your suggestion 😄

**Comment by sofietoft** [comment]
Thanks for considering the suggestion @erikjanwestendorp ! 💪 

I'll get it merged then 😄

**Pull request merged**